curriculum/challenges/english/05-apis-and-microservices/basic-node-and-express/meet-the-node-console.english.md

```diff
@@ -13,7 +13,10 @@ We recommend to keep the log panel open while working at these challenges. By re
 
 ## Instructions
 <section id='instructions'>
-Modify the <code>myApp.js</code> file to log "Hello World" to the console. 
+
+* If you have not already done so, please read the instructions in [the introduction](/learn/apis-and-microservices/basic-node-and-express/) and start a new project on Glitch using [this link](https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-express/).
+  
+* Modify the <code>myApp.js</code> file to log "Hello World" to the console. 
```

Comment:
```suggestion
* Modify the <code>myApp.js</code> file to log "Hello World" to the console. 

```

Reply:
@RandellDawson add this new line to fix the linting error.

Reply:
Dumb question, but why did it not fail when I ran `npm run test`?  How could I have caught this error before pushing back to the branch?

Reply:
I could not commit the suggestion via GitHub, but I was able to add a new commit locally and push back to the OP's branch.